src/scripts/pokemons/PokemonList.ts

```diff
@@ -14182,6 +14182,1586 @@ const pokemonList: PokemonListData[] =
+                'attack': 90,
+                'specialAttack': 80,
+                'defense': 100,
+                'specialDefense': 90,
+                'speed': 55,
+            },
+            'eggCycles': 35,
+            'levelType': LevelType.slow,
+            'exp': 177,
+            'catchRate': 45,
+        },
+        {
+            'id': 884,
+            'name': 'Duraludon',
+            'type': [
+                PokemonType.Steel,
+                PokemonType.Dragon,
+            ],
+            'base': {
+                'hitpoints': 70,
+                'attack': 95,
+                'specialAttack': 120,
+                'defense': 115,
+                'specialDefense': 50,
+                'speed': 85,
+            },
+            'eggCycles': 30,
+            'levelType': LevelType.mediumfast,
+            'exp': 187,
+            'catchRate': 45,
+        },
+        {
+            'id': 885,
+            'name': 'Dreepy',
+            'type': [
+                PokemonType.Dragon,
+                PokemonType.Ghost,
+            ],
+            'base': {
+                'hitpoints': 28,
+                'attack': 60,
+                'specialAttack': 40,
+                'defense': 30,
+                'specialDefense': 30,
+                'speed': 82,
+            },
+            'eggCycles': 40,
+            'levelType': LevelType.slow,
+            'exp': 54,
+            'catchRate': 45,
+            'evolutions': [], // TODO: Drakloak
+        },
+        {
+            'id': 886,
+            'name': 'Drakloak',
+            'type': [
+                PokemonType.Dragon,
+                PokemonType.Ghost,
+            ],
+            'base': {
+                'hitpoints': 68,
+                'attack': 80,
+                'specialAttack': 60,
+                'defense': 50,
+                'specialDefense': 50,
+                'speed': 102,
+            },
+            'eggCycles': 40,
+            'levelType': LevelType.slow,
+            'exp': 144,
+            'catchRate': 45,
+            'evolutions': [], // TODO: Dragapult
+        },
+        {
+            'id': 887,
+            'name': 'Dragapult',
+            'type': [
+                PokemonType.Dragon,
+                PokemonType.Ghost,
+            ],
+            'base': {
+                'hitpoints': 88,
+                'attack': 120,
+                'specialAttack': 100,
+                'defense': 75,
+                'specialDefense': 75,
+                'speed': 142,
+            },
+            'eggCycles': 40,
+            'levelType': LevelType.slow,
+            'exp': 300,
+            'catchRate': 45,
+        },
+        {
+            'id': 888,
+            'name': 'Zacian',
+            'type': [
+                PokemonType.Fairy,
+                PokemonType.Steel,
+            ],
+            'base': {
+                'hitpoints': 92,
+                'attack': 170,
+                'specialAttack': 80,
```

Comment:
May want to fix the stats as well, as Crowned Forms/Other forms have different stats

Reply:
Yeah, wasn't finished with remove forms